engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java

```diff
@@ -239,7 +239,7 @@ public void sendMetrics(Collection<String> metricNames)
   {
     @SuppressWarnings("FieldNameHidesFieldInSuperclass")
     private static final long serialVersionUID = 1L;
-    private OperatorMeta operatorMeta;
+    private DagNodeMeta operatorMeta;
```

Comment:
The port can belong to Operator or Module. hence used a common meta object in the portmeta. This common meta object contains only
- ports 
- name
- object

The rest of Operator specific fields are in OperatorMeta and Module specific fields are in ModuleMeta.